5g-mobility/computer-vision
# stopped persons

Feature:  Detect persons if they keep stopped for 4 frames.

How to test:

python3 track.py --source person_stopped.mov --weights ./yolov5/weights/best.pt --view-img --save-txt

Exepect results:

appear a output with field 'parado': True
